docs/content/preview/api/ysql/the-sql-language/statements/ddl_create_index.md

```diff
@@ -68,6 +68,12 @@ For details on how online index backfill works, refer to [Online Index Backfill]
 
 Regarding colocation, indexes follow their table. If the table is colocated, its index is also colocated; if the table is not colocated, its index is also not colocated.
 
+### Partitioned Indexes
+
+Creating an index on a partitioned table will automatically create a corresponding index for every partition in the default tablespace. Conversely, it is also possible to create an index on each partition individually. Prefer using the latter approach to creating partitioned indexes in the following cases:
+* Parallel writes are expected while creating the index: This is because concurrent builds for indexes on partitioned tables are not supported. In this case, it is better to use concurrent builds to create indexes on each partition individually.
```

Comment:
Blank lines around lists...
```suggestion

* Parallel writes are expected while creating the index, because concurrent builds for indexes on partitioned tables aren't supported. In this case, it's better to use concurrent builds to create indexes on each partition individually.
```